base/applications/mscutils/eventvwr/lang/ro-RO.rc

```diff
@@ -115,79 +115,79 @@ END
     EDITTEXT IDC_EDIT_EVENTS_AGE, 175, 165, 35, 14, ES_LEFT | ES_NUMBER | WS_GROUP
     CONTROL "", IDC_UPDOWN_EVENTS_AGE, UPDOWN_CLASS, UDS_SETBUDDYINT | UDS_ALIGNRIGHT |
             UDS_AUTOBUDDY | UDS_ARROWKEYS, 104, 165, 10, 14
     LTEXT "zile", IDC_STATIC, 215, 168, 20, 8
-    PUSHBUTTON "&Opțiuni implicite", IDC_RESTOREDEFAULTS, 166, 183, 70, 14
-    AUTOCHECKBOX "Conexiune de &bandă limitată", IDC_LOW_SPEED_CONNECTION, 7, 217, 167, 10
+    PUSHBUTTON "&Restaurare implicite", IDC_RESTOREDEFAULTS, 166, 183, 70, 14
+    AUTOCHECKBOX "&Folosind o conexiune de viteză mică", IDC_LOW_SPEED_CONNECTION, 7, 217, 167, 10
     PUSHBUTTON "&Curăță jurnal", ID_CLEARLOG, 195, 213, 50, 14
 END
 
 STRINGTABLE
 BEGIN
     IDS_COPYRIGHT "Drept de autor (C) 2007 Marc Piulachs (marc.piulachs@codexchange.net)"
-    IDS_APP_TITLE "Observator de evenimente"
+    IDS_APP_TITLE "Vizualizator de evenimente"
     IDS_APP_TITLE_EX "%s - %s autentificat pe \\\\"
     IDS_STATUS_MSG "%s are %lu (de) eveniment(e) (afișate: %lu)"
-    IDS_LOADING_WAIT "Jurnal de evenimente în încărcare. Așteptați…"
-    IDS_NO_ITEMS "Nu există elemente de afișat." // "No events in this log."
+    IDS_LOADING_WAIT "Se încarcă jurnalele de evenimente. Așteptați…"
+    IDS_NO_ITEMS "Nu există elemente de afișat în această vizualizare." // "No events in this log."
     IDS_EVENTLOG_SYSTEM "Jurnale de sistem"
-    IDS_EVENTLOG_APP "Jurnale de aplicație"
+    IDS_EVENTLOG_APP "Jurnale de aplicații"
     IDS_EVENTLOG_USER "Jurnale de utilizator"
     IDS_SAVE_FILTER "Jurnal de evenimente (*.evt)\0*.evt\0"
     IDS_CLEAREVENTS_MSG "Doriți să salvați acest jurnal de evenimente înainte de a-l închide?"
-    IDS_RESTOREDEFAULTS "Doriți să restaurați toate setările pentru acest jurnal la valorile lor implicite?"
-    IDS_EVENTSTRINGIDNOTFOUND "Descrierea evenimentului cu ID-ul ( %lu ) în sursa ( %s ) nu a fost găsită. Este posibil ca în calculatorul local să nu existe informațiile de registru necesare sau fișierele dll de mesaje să afișeze mesaje de la un calculator din rețea.\n\nInformații aferente evenimentului:\n\n"
+    IDS_RESTOREDEFAULTS "Doriți să restabiliți toate setările pentru acest jurnal la valorile lor implicite?"
+    IDS_EVENTSTRINGIDNOTFOUND "Imposibil de găsit descrierea evenimentului cu ID-ul ( %lu ) în sursa ( %s ). Este posibil ca computerul local să nu aibă informațiile de registru necesare sau fișierele DLL de mesaje pentru a afișa mesaje de la un calculator la distanță.\n\nUrmătoarele informații fac parte din eveniment:\n\n"
 END
 
 STRINGTABLE
 BEGIN
 /* Please note that explicit TAB characters '\t' are used to fix the alignment of the message in the MessageBox */
-    IDS_USAGE "Inspector de Evenimente ReactOS\n\
+    IDS_USAGE "Vizualizatorul de evenimente ReactOS\n\
 \n\
-EventVwr [nume calculator] [/L:<fișier jurnal de evenimente>] [/?]\n\
+EventVwr [numele calculatorului] [/L:<fișier de jurnal de evenimente>] [/?]\n\
```

Comment:
In other places, e.g. SHELL32 you structurally changed from "calculator" to "computer". Why not in this module as well? (and in other locations/matches of this PR)

Reply:
Fixed! I didn't pay attention.